### PR TITLE
PEP8 doc examples

### PR DESCRIPTION
**Pull request by QuLogic**
For some reason, I've had this sitting around for a year and a half.

Mostly a bunch of word-wrap; this actually makes the docs look better since you don't need to scroll to the right any more.

### CODE REVIEW

File: lib/iris/tests/test_coding_standards.py [hunk 1]
```diff
@@ -134,29 +134,8 @@ class StandardReportWithExclusions(pep8.StandardReport):
 
     if DOCS_DIRS:
         expected_bad_docs_files = [
-            '*/example_code/General/SOI_filtering.py',
```
Comment (pelson) [inline lib/iris/tests/test_coding_standards.py]: :+1: :100:

**Comment by pelson** [comment]
The failing travis test: 

```
-------------------- >> begin captured stdout << ---------------------
/home/travis/build/SciTools/iris/docs/iris/src/userguide/plotting_examples/1d_with_legend.py:26:37: E202 whitespace before ')'
/home/travis/build/SciTools/iris/docs/iris/src/userguide/plotting_examples/1d_with_legend.py:28:1: E265 block comment should start with '# '
```

**Comment by QuLogic** [comment]
Oh, I missed double-checking that directory on rebase; should be good now.

**Comment by rhattersley** [comment]
Thanks @QuLogic :+1:

**Pull request merged**